app/services/response_file_builder/build_response_pdf_file.rb

```diff
@@ -56,127 +63,153 @@ def pdf_fields
+      apply_field result, response.additional_information_rtf_file?, :header, :rtf
     end
 
     def apply_claimant_pdf_fields(result)
-      result['1.1'] = response.claimants_name
+      apply_field result, response.claimants_name, :claimant, :claimants_name
     end
 
     def apply_respondent_pdf_fields(result)
       respondent = response.respondent
       address = respondent.address
-      result['2.1'] = respondent.name
-      result['2.2'] = respondent.contact
-      result['2.3 number or name'] = address.building
-      result['2.3 street'] = address.street
-      result['2.3 town city'] = address.locality
-      result['2.3 county'] = address.county
-      result['2.3 postcode'] = address.post_code.tr(' ', '')
-      result['2.3 dx number'] = respondent.dx_number
-      result['2.4 phone number'] = respondent.address_telephone_number
-      result['2.4 mobile number'] = respondent.alt_phone_number
-      result['2.5'] = respondent.contact_preference || 'Off'
-      result['2.6 email address'] = respondent.email_address
-      result['2.6 fax number'] = respondent.fax_number
-      result['2.7'] = respondent.organisation_employ_gb
-      result['2.8'] = respondent.organisation_more_than_one_site ? 'yes' : 'no'
-      result['2.9'] = respondent.employment_at_site_number
+      apply_field result, respondent.name, :respondent, :name
+      apply_field result, respondent.contact, :respondent, :contact
+      apply_field result, address.building, :respondent, :address, :building
+      apply_field result, address.street, :respondent, :address, :street
+      apply_field result, address.locality, :respondent, :address, :locality
+      apply_field result, address.county, :respondent, :address, :county
+      apply_field result, address.post_code.tr(' ', ''), :respondent, :address, :post_code
+      apply_field result, respondent.dx_number, :respondent, :address_dx_number
+      apply_field result, respondent.address_telephone_number, :respondent, :phone_number
+      apply_field result, respondent.alt_phone_number, :respondent, :mobile_number
+      apply_field result, respondent.contact_preference, :respondent, :contact_preference
+      apply_field result, respondent.email_address, :respondent, :email_address
+      apply_field result, respondent.fax_number, :respondent, :fax_number
+      apply_field result, respondent.organisation_employ_gb, :respondent, :employ_gb
+      apply_field result, respondent.organisation_more_than_one_site, :respondent, :multi_site_gb
+      apply_field result, respondent.employment_at_site_number, :respondent, :employment_at_site_number
     end
 
     def apply_acas_pdf_fields(result)
-      result['new 3.1'] = response.agree_with_early_conciliation_details ? 'Yes' : 'No'
-      result['new 3.1 If no, please explain why'] = response.disagree_conciliation_reason
+      apply_field result, response.agree_with_early_conciliation_details?, :acas, :agree
```

Comment:
Instead of hard coding the field name, it is now calculated from within apply_field - all it needs to know is the path of the value from the yaml file (in this case "acas.agree")